treeherder/perf/alerts.py

```diff
@@ -110,5 +116,6 @@ def generate_new_alerts_in_series(signature):
                     amount_abs=alert_properties.delta,
                     prev_value=prev_value,
                     new_value=new_value,
-                    t_value=t_value)
+                    t_value=t_value,
+                    title=title)
```

Comment:
I would calculate the name inside the PerformanceAlert model itself (inside the "save" method, which is already defined), that way if another caller creates a PerformanceAlert it will automatically get a title (maybe let the user optionally pass in their own title optionally, so only auto-create one if not supplied)